src/client/containers/BoardPageComponents/BoardAddCandidate/BoardAddCandidate.js

```diff
@@ -3,43 +3,60 @@ import './BoardAddCandidate.style.css';
 import Input from '../../../components/InputComponent/InputComponent';
 import AddButton from '../../../components/AddButton/AddButton.component';
 import AddCandidate from './AddCandidate';
-import { useUser } from '../../firebase/UserContext';
+import { useUser } from '../../../firebase/UserContext';
+import { useBoard } from '../../BoardPage/BoardProvider';
 
 export default function BoardAddCandidate() {
   const [newCandidateName, setNewCandidateName] = useState('');
   const { user } = useUser();
   const userId = user[0].id;
+  const { boardInfo } = useBoard();
+  const { setBoardLoading } = useBoard();
+  const deadlineDate = new Date(boardInfo.deadline);
+  const today = new Date();
 
   const newCandidate = {
-    boardId: 1,
+    boardId: boardInfo.id,
     name: newCandidateName,
     isBlocked: false,
   };
 
   const saveCandidate = (e) => {
     e.preventDefault();
-    AddCandidate(newCandidate, userId, newCandidate.boardId);
+    AddCandidate(newCandidate, userId, newCandidate.boardId, setBoardLoading);
     setNewCandidateName('');
   };
 
   return (
     <div>
-      <div className="Input-component">
-        <Input
-          type="text"
-          placeholder="Add candidate..."
-          theme="dark"
-          borderShape="curved"
-          inputValue={newCandidateName}
-          onChange={setNewCandidateName}
-        />
-        <AddButton
-          className="add-button"
-          type="button"
-          disabled={false}
-          onClick={saveCandidate}
-        />
-      </div>
+      {today > deadlineDate ? (
```

Comment:
I think the component should just not be shown if the its past the deadline. So in its parent component you would just show the component if its before the deadline

Reply:
Or maybe show the disabled add button when past the deadline

Reply:
What do you think?

Reply:
I’m agree with you. I think it shouldn’t be shown after deadline, I just tried to follow the model in figma. 
![809B22B6-E15E-45A4-8AE7-1FCBCD612F6E](https://user-images.githubusercontent.com/75580873/130467738-42f200ed-1d69-482b-8ef9-e4157783e43d.png)

Reply:
> I’m agree with you. I think it shouldn’t be shown after deadline, I just tried to follow the model in figma.
> ![809B22B6-E15E-45A4-8AE7-1FCBCD612F6E](https://user-images.githubusercontent.com/75580873/130467738-42f200ed-1d69-482b-8ef9-e4157783e43d.png)

As far as I can see the component is still showing after the deadline, right. The grey candidates means past deadline, right?

Reply:
@Hani-far 
So which is it? Should the component be shown after deadline or not?

Reply:
according to the figma it should be shown after the deadline as well but the color of the input and add button will be gray and add button will be disabled.

Reply:
Thats fine by me. Is it doing that now?

Reply:
I have done it in this PR by checking the deadline and returned the disabled component if the deadline passed.

Reply:
Yes, It is. I tested it on the Board page